plugins/postcss-design-tokens/README.md

```diff
@@ -0,0 +1,259 @@
+# PostCSS Design Tokens [<img src="https://postcss.github.io/postcss/logo.svg" alt="PostCSS Logo" width="90" height="90" align="right">][postcss]
+
+[<img alt="npm version" src="https://img.shields.io/npm/v/@csstools/postcss-design-tokens.svg" height="20">][npm-url] [<img alt="Build Status" src="https://github.com/csstools/postcss-plugins/workflows/test/badge.svg" height="20">][cli-url] [<img alt="Discord" src="https://shields.io/badge/Discord-5865F2?logo=discord&logoColor=white">][discord]
+
+[PostCSS Design Tokens] lets you use design tokens in your CSS source files.
+
+```json
+{
+	"color": {
+		"background": {
+			"primary": { "value": "#fff" }
+		}
+	},
+	"size": {
+		"spacing": {
+			"small": { "value": "16px" }
+		}
+	}
+}
+```
+
+```pcss
+@design-tokens url('./tokens.json') format('style-dictionary3');
+
+.foo {
+	color: design-token('color.background.primary');
+	padding-left: design-token('size.spacing.small' to px);
```

Comment:
Is it possible to use `size.spacing.small` without `to`? If so, should we illustrate?

Reply:
Yes and done!